__TESTS__/unit/actions/Condition.test.ts

```diff
@@ -0,0 +1,117 @@
+import Condition, {fromString} from '../../../src/actions/condition/Condition';
+import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
+import TransformableImage from "../../../src/transformation/TransformableImage";
+import {variable} from "../../../src/actions/variable/Variable";
+
+const CONFIG_INSTANCE = new CloudinaryConfig({
+  cloud: {
+    cloudName: 'demo'
+  }
+});
+
+describe('Tests for Transformation Action -- Condition', () => {
+  it('Ensures Condition namespace is accepted as an action to TransformableImage', () => {
+    const tImage = new TransformableImage();
+    // Ensures it compiles and doesn't throw
+    expect(
+      tImage.ifCondition(Condition.fromString('ar >= 1.0'))
+    ).toEqual(tImage);
+  });
+
+  it('Creates a cloudinaryURL with condition =', () => {
+    const url = new TransformableImage()
+      .setConfig(CONFIG_INSTANCE)
+      .ifCondition(Condition.fromString('ar = 1.0'))
+      .setPublicID('sample')
+      .toURL();
+
+    expect(url).toBe('http://res.cloudinary.com/demo/image/upload/if_ar_eq_1.0/sample');
```

Comment:
This transformation isn't valid, can we make sure our tests have valid and correct usages?
(this, for example, is missing an end-if)

Tests become a part of our informal documentation as to what's allowed and what's not

Reply:
updated